hadoop-ozone/dist/src/shell/ozone/ozone

```diff
@@ -105,8 +105,9 @@ function ozonecmd_case
       # Corresponding Ratis issue https://issues.apache.org/jira/browse/RATIS-534.
       # TODO: Fix the problem related to netty resource leak detector throwing
       # exception as mentioned in HDDS-3812
-      HDDS_DN_OPTS="-Dorg.apache.ratis.thirdparty.io.netty.allocator.useCacheForAllThreads=false -Dorg.apache.ratis.thirdparty.io.netty.leakDetection.level=disabled -Dlog4j.configurationFile=${HADOOP_CONF_DIR}/dn-audit-log4j2.properties ${HDDS_DN_OPTS}"
-      HADOOP_OPTS="${HADOOP_OPTS} ${HDDS_DN_OPTS}"
+      hadoop_deprecate_envvar HDDS_DN_OPTS OZONE_DATANODE_OPTS
+      OZONE_DATANODE_OPTS="-Dorg.apache.ratis.thirdparty.io.netty.allocator.useCacheForAllThreads=false -Dorg.apache.ratis.thirdparty.io.netty.leakDetection.level=disabled -Dlog4j.configurationFile=${HADOOP_CONF_DIR}/dn-audit-log4j2.properties ${OZONE_DATANODE_OPTS}"
+      HADOOP_OPTS="${HADOOP_OPTS} ${OZONE_DATANODE_OPTS}"
```

Comment:
I think this line is not necessary, because it will be added by 

```
hadoop_subcommand_opts "${HADOOP_SHELL_EXECNAME}" "${HADOOP_SUBCMD}"
```

But please check it.

Reply:
Yes looks like `HADOOP_OPTS="${HADOOP_OPTS} ${OZONE_DATANODE_OPTS}"` is redundant with `hadoop_subcommand_opts` since we are aligning the OPTS names.